src/api/endpoints/index.ts

```diff
@@ -117,17 +118,51 @@ export const getCuries = async (term) => {
 }
 
 export const getMatchTerms = async (term, filters = {}) => {
-  const {  getMatchTerms } = useMockApi();
-
-  /** Call Endpoint */
-  return getMatchTerms("base", term, filters).then((data) => {
-      return termParser(data, term, filters);
-    })
-    .catch((error) => {
-      return error;
+  const proxyUrl = 'http://localhost:3000/olympianGods';
```

Comment:
factor out this in a configuration file so that then we will have to change it only once please. Thx